web/tests/upload.test.js

```diff
@@ -611,6 +611,12 @@ test("uppy_events", ({override_rewire, mock_template}) => {
     };
     on_info_visible_callback();
 
+    let hide_upload_banner_called = false;
```

Comment:
This pattern is used elsewhere in the file with `hide_upload_banner`, but I would love some guidance as to whether its identical use here is warranted--and how this might be improved.

Reply:
It's not the best pattern, in that you have to remember to assert or reset it to `false` before each test that will check it, or your test could end up not verifying what you think it is.

But it's probably OK for this component.

We're also trying to avoid using `override_rewire` as it is a bit evil, but this test module at least is already hopelessly contaminated, so not worth thinking about in this PR.